public/css/adhunik.css

```
@@ -0,0 +1,915 @@
+    left: 0;
+    right: 0;
+    padding: 1.5rem 3rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    z-index: 100;
+    background: linear-gradient(to bottom, var(--void) 0%, transparent 100%);
+}
+
+.logo {
+    font-family: 'Orbitron', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Orbitron" (font-family-name-quotes)

(font-family-name-quotes)